### PR TITLE
[PREVIEW] add edit functionality for listing request

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/app/hearing-part/components/listing-create/listing-create.component.spec.ts [hunk 1]
```diff
@@ -232,24 +253,24 @@ describe('ListingCreateComponent', () => {
...
       });
 
-      it('with some notes it should set default notes post-creation', () => {
-        component.listing.notes = [{...note, content: 'custom content'}];
-
-        component.create();
-
-        expect(component.listing.notes).toEqual(listingCreateNoteConfig.defaultNotes());
-    });
+    //   it('with some notes it should set default notes post-creation', () => {
```
Comment (zielinskikamil) [inline src/app/hearing-part/components/listing-create/listing-create.component.spec.ts]: just remember about removing this comments ;)

File: src/app/hearing-part/components/listing-create/listing-create.component.ts [hunk 2]
```diff
@@ -86,37 +124,66 @@ export class ListingCreateComponent implements OnInit {
         return targetFrom.isSameOrBefore(targetTo) ? null : { targetFromAfterTargetTo: true };
     }
 
+    private setNotesIfExist(hp: ListingCreate) {
```
Comment (zielinskikamil) [inline src/app/hearing-part/components/listing-create/listing-create.component.ts]: hp: stands from what ?:)

File: src/app/sessions/containers/sessions-create/sessions-create.component.ts [hunk 3]
```diff
@@ -43,11 +45,17 @@ export class SessionsCreateComponent implements OnInit {
...
     create(session) {
+        this.sessionId = session.id;
         this.sessionCreationService.create(session);
-        this.dialogRef = this.openDialog(session);
+        this.dialogRef = this.openDialog();
+        this.dialogRef.afterClosed().subscribe(() => this.getCreatedSession(this.sessionId));
     }
 
-    private openDialog(session) {
+    public getCreatedSession(sessionId: string) {
```
Comment (zielinskikamil) [inline src/app/sessions/containers/sessions-create/sessions-create.component.ts]: name here is confusing, get indicates that you will return sth, but you just dispatch an action

File: src/app/hearing-part/components/listing-create/listing-create.component.ts [hunk 4]
```diff
@@ -57,25 +77,47 @@ export class ListingCreateComponent implements OnInit {
         this.store.dispatch(new JudgeActions.Get());
     }
 
+    save() {
```
Comment (filipczyks) [inline src/app/hearing-part/components/listing-create/listing-create.component.ts]: we could probably use inheritance instead of this 'mode' idea

**Pull request merged**